agimus_controller_ros/package.xml

```diff
@@ -24,7 +43,16 @@
     <!-- Group for interface packages -->
   <member_of_group>rosidl_interface_packages</member_of_group>
 
+  <!-- LAAS dependencies. -->
+  <exec_depend>pinocchio</exec_depend>
+  <exec_depend>example-robot-data</exec_depend> <!-- install manually. -->
```

Comment:
I don't think we need this as a dependency. All urdfs will come from different sources